src/resources/postcss/rsvp/_actions.pcss

```diff
@@ -0,0 +1,112 @@
+/**
+ * Event Tickets - RSVP Structure
+ *
+ * @since TDB
+ */
+
+.event-tickets {
+
+	.tribe-tickets__rsvp-actions-wrapper {
+		flex: none;
+		display: flex;
+		justify-content: center;
+		text-align: center;
+		align-items: center;
+
+		@media(--viewport-medium) {
+			border-left: 1px dashed var(--color-border-secondary);
+			width: calc(3 * var(--grid-width-1-of-8));
+		}
+	}
+
+	.tribe-tickets__rsvp-actions {
+		border-top: 1px dashed var(--color-border-secondary);
+		margin-top: var(--spacer-6);
+		padding: var(--spacer-6) 0;
+		width: 100%;
+
+
+		@media(--viewport-medium) {
+			border-top: 0;
+			margin-top: 0;
+		}
+
+		.tribe-common-c-btn {
+			max-width: 125px;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * RSVP Actions (Going / Can't go)
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-tickets__rsvp-actions-rsvp-going {
+		margin-top: var(--spacer-2);
+	}
+
+	.tribe-tickets__rsvp-actions-rsvp-not-going {
+		margin-top: var(--spacer-2);
+	}
```

Comment:
Right! the thing is that they're two different blocks, it's not that it's the same blocks with a different variation (modifier). The `-going` is wrapping the things that we have for the "going" button and all that. The same with the "not going". And one block comes after the other.

Reply:
Is the structure the same? Could you make it so that the CSS follows the structure as well? If it doesn't make sense, no worries, just asking the questions so we consider it.

Reply:
Well, for now yes. It's kind of two "rows" one on top of the other. <img src="https://cldup.com/EHIBv9T_zd.gif">